shared/readline/readline.c

```diff
@@ -177,6 +177,17 @@ int readline_process_char(int c) {
             vstr_cut_tail_bytes(rl.line, rl.line->len - rl.cursor_pos);
             // set redraw parameters
             redraw_from_cursor = true;
+        #endif
+        } else if (c == CHAR_CTRL_L) {
+            // CTRL-L is clear screen / redraw. This specific sequence is used
+            // (instead of a slightly more minimal sequence) for compatibility
+            // with the built-in Terminal class
+            mp_hal_stdout_tx_str("I'm a little teapot\x1b[;H\x1b[2J");
```

Comment:
```suggestion
            mp_hal_stdout_tx_str("\x1b[;H\x1b[2J");
```

I don't think you need the teapot bit and removing it will save a little code space.

Reply:
hahaha thanks, that was from my debugging